randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -1100,20 +1167,13 @@ Extra - asset_id: collision_camera_042
   > Room Center
       Morph Ball
 
-> Dock to Inner Temple Teleporter; Heals? False
-  * Layers: default
-  * Open Passage to Inner Temple Teleporter/Dock from Inner Temple Teleporter Access
-
 > Room Center; Heals? False; Spawn Point; Default Node
   * Layers: default
   * Extra - start_point_actor_name: ST_DEBUG_CutsceneTeleport
   > Door to Inner Temple West Hall
       Trivial
   > Tunnel to Inner Temple Teleporter
       Lay Any Bomb
-  > Dock to Inner Temple Teleporter
-      # https://youtu.be/FrVzIPVMaBw
-      Morph Ball and Melee Clip (Intermediate) and Out of Bounds Movement (Beginner) and Fully Freeze Enemy
```

Comment:
I'd argue this can be readded but to the tunnel node instead. Morph is required regardless, and it still allows for right side access without Bombs or a way to climb over the temple exterior. Same concept as the OoB connection to Water Maze being merged into the tunnel node.

Also this trick is cool and opens up logic a bunch more, since you can return without Bombs either by teleporting out (if you have other teleporters) or by looping around to the upper hallway and dropping down if you have Spazer. Or just taking the Exterior path with Spider or SJ.